.github/CONTRIBUTING.md

```diff
@@ -196,3 +196,40 @@ promptly to pull requests, particularly if they do not relate to an existing
 GitHub issue where the maintainer team has already participated. We _are_
 grateful for all contributions however, and will give feedback on pull requests
 as soon as we're able to.
+
+## Releasing
+
+Releases are made on a reasonably regular basis by the maintainers (HashiCorp staff), using our internal tooling. The following notes are only relevant to maintainers.
+
+Release process:
+
+ 1. Update [`version/VERSION`](https://github.com/hashicorp/terraform-ls/blob/main/version/VERSION) to remove `-dev` suffix and set it to the intended version to be released
+ 1. Wait for [`build` workflow](https://github.com/hashicorp/terraform-ls/actions/workflows/build.yml) to finish
```

Comment:
We also need to wait until the `prepare` workflow finishes. That one automatically starts after `build`.